samples/crypto/psa_tls/overlays/cc3xx-oberon-psa.conf

```diff
@@ -1,8 +1,3 @@
 CONFIG_NRF_SECURITY=y
 CONFIG_PSA_CRYPTO_DRIVER_CC3XX=y
 CONFIG_PSA_CRYPTO_DRIVER_OBERON=y
-
-CONFIG_PSA_CRYPTO_DRIVER_ALG_STREAM_CIPHER_OBERON=y
```

Comment:
I'm sure these were added to ensure AES 256 support for CC310 devices as it is commonly used for TLS/DTLS handshake... How is this handled in the new scheme?

Reply:
Minor correction,  it is handled by "PSA_WANT_AES_KEY_SIZE_256", that's the user selectable option. There is no such symbol as PSA_ACCEL_AES_KEY_SIZE_256 for now. The key sizes in Oberon are handled with the symbols which are algorithm related (e.g PSA_ACCEL_AES128_CCM).